luminoth/tools/cloud/gcloud.py

```diff
@@ -232,13 +263,96 @@ def train(job_id, service_account_json, bucket_name, region, config_files,
         click.echo('Job {} submitted successfully.'.format(job_id))
         click.echo('state = {}, createTime = {}'.format(
             res.get('state'), res.get('createTime')))
+
+        save_run(config, environment='gcloud', extra_config=job_spec)
+
     except Exception as err:
         click.echo(
             'There was an error creating the training job. '
             'Check the details: \n{}'.format(err._get_reason())
         )
 
 
+@gc.command(help='Start a evaluation job')
+@click.option('--job-id', required=True, help='JobId for saving models and logs.')  # noqa
+@click.option('--service-account-json', required=True)
+@click.option('--bucket', 'bucket_name', help='Where to save models and logs.')  # noqa
+@click.option('dataset_split', '--split', default='val', help='Dataset split to use.')  # noqa
+@click.option('--region', default='us-central1', help='Region in which to run the job.')  # noqa
+@click.option('--scale-tier', default=DEFAULT_SCALE_TIER, type=click.Choice(SCALE_TIERS))  # noqa
+@click.option('--rebuild', default=False, is_flag=True, help='Rebuild and upload package.')  # noqa
+@click.option('--postfix', default='eval', help='Postfix for the evaluation job name.')  # noqa
+def evaluate(job_id, service_account_json, bucket_name, dataset_split, region,
+             scale_tier, rebuild, postfix):
+    project_id = get_project_id(service_account_json)
+    if project_id is None:
+        raise ValueError(
+            'Missing "project_id" in service_account_json "{}"'.format(
+                service_account_json))
+
+    if bucket_name is None:
+        client_id = get_client_id(service_account_json)
+        bucket_name = 'luminoth-{}'.format(client_id)
+        click.echo(
+            'Bucket name not specified. Using "{}".'.format(bucket_name))
+
+    credentials = get_credentials(service_account_json)
+    validate_region(region, project_id, credentials)
+    job_folder = 'lumi_{}'.format(job_id)
+
+    if rebuild:
+        bucket = get_bucket(service_account_json, bucket_name)
+        build_package(bucket, job_folder)
+
+    job_dir = 'gs://{}/{}'.format(bucket_name, job_folder)
+
+    config_path = '{}/{}'.format(job_dir, DEFAULT_CONFIG_FILENAME)
+    package_dir = '{}/{}'.format(job_dir, DEFAULT_PACKAGES_PATH)
+
+    package_files = tf.gfile.ListDirectory(package_dir)
+    package_filename = [n for n in package_files if n.endswith('tar.gz')][0]
+    package_path = '{}/{}'.format(package_dir, package_filename)
+
+    cloudml = cloud_service(credentials, 'ml')
+
+    args = [
+        ['--config', config_path],
+        ['--split', dataset_split],
+    ]
+
+    training_inputs = {
+        'scaleTier': scale_tier,
+        'packageUris': [package_path],
+        'pythonModule': 'luminoth.eval',
+        'args': args,
+        'region': region,
+        'jobDir': job_dir,
+        'runtimeVersion': '1.2'
```

Comment:
Use external constant to avoid having to change it twice (or more) once the new version is released.

Reply:
Great catch!